File: Java/1015.java
import java.util.Locale;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Locale.setDefault(Locale.US);

        Scanner input = new Scanner(System.in);

        double x1, x2, y1, y2, distancia;

        x1 = input.nextDouble();
        y1 = input.nextDouble();
        x2 = input.nextDouble();
        y2 = input.nextDouble();

        distancia = Math.sqrt(Math.pow(x2 - x1, 2) + Math.pow(y2 - y1, 2));

        System.out.printf("%.4f\n", distancia);

    }
}

File: Python/1015.py
x1 , y1 = list(map(float,input().split()))
x2 , y2 = list(map(float,input().split()))

distacia =( (x2 - x1)**2 + (y2 - y1)**2)**(1/2)

print (f"{distacia:.4f}")


